lib/kano-register/kano-email-field.js

```diff
@@ -0,0 +1,96 @@
+var xtag = window.xtag;
+
+module.exports = function () {
+    xtag.register('kano-email-field', {
+        content: function () {/*
+            <div class="kano-email-field">
```

Comment:
This is a way to do multiline now. The other solution would be multiline concat. We have nothing better until we use es6 template string. An other solution would be to store the template in a variable and tell gulp to get the content of the template and set it into the variable.